docs/csharp/programming-guide/concepts/linq/serializing-to-an-xmlreader-invoking-xslt.md

```diff
@@ -55,4 +55,4 @@ Console.WriteLine(newTree);
   
 ## See also
 
-- [Serializing XML Trees (C#)](../../../../csharp/programming-guide/concepts/linq/serializing-xml-trees.md)
+- [Serializing XML Trees (C#)](../../../../csharp/programming-guide/concepts/linq/preserving-white-space-while-serializing.md)
```

Comment:
See my earlier comment about linking to the Files, TextWriters, and XMLWriters topic.